packages/story-editor/src/app/media/utils/useMediaUploadQueue/reducer.js

```diff
@@ -271,6 +274,62 @@ export function finishMuting(state, { payload: { id, file } }) {
   };
 }
 
+/**
+ * Starts trimming a file.
+ *
+ * @param {Object} state Current state.
+ * @param {Object} action Action object.
+ * @param {Object} action.payload Action payload.
+ * @param {string} action.payload.id Item ID.
+ * @return {Object} New state
+ */
+export function startTrimming(state, { payload: { id } }) {
```

Comment:
These should have tests like all the other reducers

Reply:
Good catch. Added here - https://github.com/google/web-stories-wp/pull/8972/commits/ff4cd935b4dbd6459de2bcf7cd948a7834face71